tests/models/test_headers.py

```diff
@@ -13,6 +13,7 @@ def test_headers():
     assert h["a"] == "123, 456"
     assert h.get("a") == "123, 456"
     assert h.get("nope", default=None) is None
+    assert h.getlist("a") == ["123", "456"]
```

Comment:
Do we want a `with pytest.warns(DeprecatedWarning)` here, so that these known warnings don't show up in the pytest output?

Reply:
💯 - Yes, thanks! Which helped catch that they needed to be `DeprecationWarning` in line with everything else (not PendingDeprecationWarning).